ereolen_support/ereolen_material.form.inc

```diff
@@ -0,0 +1,88 @@
+<?php
+
+/**
+ * @file
+ * Add eReolen material selection form.
+ */
+
+include_once 'ereolen_support.helper.inc';
+
+/**
+ * Create material selection form.
+ *
+ * @param array $form
+ *   The form to create.
+ * @param array $form_state
+ *   The state of the form.
+ *
+ * @return mixed
+ *
+ * @throws \Exception
+ */
+function ereolen_material_form($form, &$form_state) {
+  $form['#attributes'] = [
+    'name' => 'ereolen_material_form',
+    'class' => ['node-webform'],
+    'onsubmit' => 'op.disabled = true; return true;'
```

Comment:
Is this a known Drupal trick? Is `op` defined globally?

Reply:
I'm actually not sure where it's from it's a copy from ereolen_support_form but where that is fetched from i don't know. However with honeypot installed i don't think we need it anymore.